test/parallel/test-buffer-copy.js

```diff
@@ -234,3 +235,12 @@ assert.deepStrictEqual(c, b.slice(0, c.length));
   // No copying took place:
   assert.deepStrictEqual(c.toString(), 'C'.repeat(c.length));
 }
+
+
+// Test its possible to copy buffer with length > 2^32
```

Comment:
Ditto

Reply:
Comment is replaced with a reference to the [issue](https://github.com/nodejs/node/issues/55422) as of commit [5ca703f](https://github.com/nodejs/node/pull/55492/commits/5ca703f01e7aa41eef7e3c30ac821303293cd019).